### PR TITLE
🏗 Report test status for integration tests on stable browsers on SauceLabs

### PR DESCRIPTION
**Pull request by danielrozenberg**
Wrapping up a dangling TODO from PR #21863

### CODE REVIEW

**Comment by danielrozenberg** [comment]
Results check out ^__^

File: build-system/tasks/runtime-test/index.js [hunk 1]
```diff
@@ -400,9 +404,24 @@ async function runTests() {
...
+    let totalStableFailed = 0;
+    const partialTestRunComplete = async(browsers, results) => {
+      if (results.error) {
+        // TODO(danielrozenberg): the last batch fails with an error because of
+        // IE 11 failed tests. Uncomment this once there issues are resolved.
+        // await reportTestErrored();
+      } else {
+        totalStableSuccess += results.success;
+        totalStableFailed += results.failed;
+      }
```
Comment (rsimha) [inline build-system/tasks/runtime-test/index.js]: Will the above block work if there's another browser with passing tests in the same batch as IE 11? Or will it ignore the count for that browser? For now, would it make sense to remove the `if` block?

File: build-system/tasks/runtime-test/index.js [hunk 2]
```diff
@@ -432,9 +451,13 @@ async function runTests() {
    * @param {string} batchName a human readable name for the batch.
    * @param {!Array{string}} browsers list of SauceLabs browsers as
    *     customLaunchers IDs.
+   * @param {!Function} runComplete a function to execute on the `run_complete`
```
Comment (rsimha) [inline build-system/tasks/runtime-test/index.js]: nit: For clarity, rename this param to `runCompleteFn`. (I was confused until I saw this line.)

File: build-system/tasks/runtime-test/index.js [hunk 3]
```diff
@@ -413,7 +432,7 @@ async function runTests() {
 
     if (browsers.beta.length) {
       const allBatchesExitCodes = await runTestInBatchesWithBrowsers(
-          'beta', browsers.beta);
+          'beta', browsers.beta, /* runComplete */ () => {});
```
Comment (estherkim) [inline build-system/tasks/runtime-test/index.js]: Nit: do you want to add the 260 beta tests to the sum? That's a total 650 tests which is pretty cool

File: build-system/tasks/runtime-test/index.js [hunk 4]
```diff
@@ -400,9 +404,24 @@ async function runTests() {
             ? 'beta' : 'stable']
           .push(browserId);
     }
+
+    let totalStableSuccess = 0;
+    let totalStableFailed = 0;
+    const partialTestRunComplete = async(browsers, results) => {
+      if (results.error) {
+        // TODO(danielrozenberg): the last batch fails with an error because of
```
[the remark this answers is not in the document]
Comment (rsimha) [reply build-system/tasks/runtime-test/index.js]: It runs zero tests, which is getting flagged as an error. Adding just one opt-in IE test will solve this.

File: build-system/tasks/runtime-test/index.js (hunk 3 above)
Comment (rsimha) [reply build-system/tasks/runtime-test/index.js]: @danielrozenberg is going to generate a separate non-blocking report for tests on beta browsers.

File: build-system/tasks/runtime-test/index.js (hunk 2 above)
Comment (danielrozenberg) [reply build-system/tasks/runtime-test/index.js]: Done

File: build-system/tasks/runtime-test/index.js (hunk 1 above)
Comment (danielrozenberg) [reply build-system/tasks/runtime-test/index.js]: Good catch. I'll remove the if clause and replace it with a long comment that explains what the behaviour is and what it should be ideally

File: build-system/tasks/runtime-test/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (danielrozenberg) [reply build-system/tasks/runtime-test/index.js]: What @rsimha said. @estherkim keep me posted when you see the first opted-in IE 11 test!

File: build-system/tasks/runtime-test/index.js (hunk 3 above)
Comment (danielrozenberg) [reply build-system/tasks/runtime-test/index.js]: Yep. Right now reporting even one failed tests fails the whole shard. I need to modify the test-status app to allow reporting "flaky" test types (for saucelabs-beta and for e2e)

File: build-system/tasks/runtime-test/index.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (rsimha) [reply build-system/tasks/runtime-test/index.js]: You can include an IE test in this PR by adding `.enableIe()` to the end of a `describe` or `it` block. Pick one of the tests that were [originally enabled](https://github.com/ampproject/amphtml/pull/11030/files#diff-24fc0abb04a59940a337b4a27485ab05) for IE.

[the remark this answers is not in the document]
Comment (danielrozenberg) [reply build-system/tasks/runtime-test/index.js]: Unfortunately this specific test doesn't pass on IE. I'll work with @estherkim to figure out when we can turn one on. No rush with this PR in any case

[the remark this answers is not in the document]
Comment (rsimha) [reply build-system/tasks/runtime-test/index.js]: SGTM.

/cc @lannka who was looking to enable IE tests in #22183.

**Comment by danielrozenberg** [comment]
It finally works!! https://travis-ci.org/ampproject/amphtml/jobs/539283124#L2095
Thanks @estherkim for fixing the IE-11 issue in #22557

**Comment by estherkim** [comment]
Awesome!! :)

**Pull request merged**